articles/Frontend/events-model.md

```diff
@@ -0,0 +1,410 @@
+# Events model
```

Comment:
Missing tags, not sure which one of the suggestion below I would use, possibly both

<!-- 
tags:
       JavaScript Components:
           - "Calculator whatever this part is"
       Calculator:
           - "whatever this part is"
-->

Reference: https://hub.taotesting.com/taohubdocs/tags-and-tag-groups#tags-in-groups

Missing mandatory synopsis

> This article explains...